### ISSUE TITLE
Fails to function

### ISSUE THREAD

**Issue by aadityadamani** [Goteo/Goteo#160]
I have installed Goteo in my wamp root directory. Everything works fine, but post the landing page if I try to login, create project or navigate anywhere else it shows error. Kindly help.

I have defined my SITE_URL as : 

define('SITE_URL', 'http://localhost/'); // endpoint url
define('SRC_URL', 'http://localhost/');  // host for statics
define('SEC_URL', 'http://localhost/');  // with SSL certified

![image](https://cloud.githubusercontent.com/assets/12599254/7800231/0f0a6012-0330-11e5-8493-beef3d70b34d.png)

![image](https://cloud.githubusercontent.com/assets/12599254/7800251/50671186-0330-11e5-8de0-1188999f537a.png)

**Comment by aadityadamani** [comment]
Now the CSS is bugged.

![image](https://cloud.githubusercontent.com/assets/12599254/8003134/78a04266-0b92-11e5-8e77-72878351594d.png)

**Comment by cesc1989** [comment]
You need to check:
- Apache mod_rewrite module is enabled
- You have the project .htaccess
- The apache conf file allows the .htaccess file works.

**Comment by aadityadamani** [comment]
Yes mod_rewrite module is enabled, I do have the project .htaccess and the apache conf file allows the .htaccess too. Why does the header and footer not detect the css? As shown in the screenshot above.

**Comment by cesc1989** [comment]
It's pretty weird now @aadityadamani What errors the console displays?

Are you still facing the **not found** error or just the css not loading?

**Comment by aadityadamani** [comment]
Just the css not loading. I tried couple of fresh installs. Im working on localhost.

**Comment by muzudho** [comment]
Hi.
I guess. I delete slash of end.

[before]
define('SITE_URL', 'http://localhost/'); // endpoint url
define('SRC_URL', 'http://localhost/'); // host for statics
define('SEC_URL', 'http://localhost/'); // with SSL certified

[after]
define('SITE_URL', 'http://localhost'); // endpoint url
define('SRC_URL', 'http://localhost'); // host for statics
define('SEC_URL', 'http://localhost'); // with SSL certified
